internal/queue/dejq/worker.go

```diff
@@ -0,0 +1,80 @@
+package dejq
```

Comment:
Oh need to find a better name for this package, there is no `dejq` dependency anymore, will rename in the next push. This is basically the "implementation" package, there is another one called "stub" which is used in unit tests (does nothing).

Reply:
Why is this separate from the `pkg/worker`? I thought that package could have implementation as well, right?

Reply:
Yeah good question, the reason is that I wanted the `pkg/worker` package to be really clean. The `internal/queue` package contains two subpackages (stub and impl) and one or another are imported (in unit tests/in normal code). On top of that, it contains the app-dependent configuration, handler registration and initialization code. This cannot be in the `pkg` package because it depends on internal stuff.

It was your idea to put it into `pkg`, tho, I can move it into `internal` and put everything under one roof: registration of job handlers, configuration, initialization and stub (no operation implementation). Then it could live all in a single package. Honestly, I don't see other project using our very simple job queue and I would probably just copy-paste it if I was starting a new project with the same requirements. So I am all for it, it can be confusing to have two packages: worker and queue.

Reply:
oki, I'm good with that, it was just an idea, but I do not insist on the `pkg`, you're right that it's not too likely someone would use it :)

My main point here waas: Can we pls rename it from `queue/dejq` to `queue/impl`? :)

Reply:
Absolutely, I want the `dejq` name to disappear, it was unpronouncable. What a bad name.